autoscheduler/frontend/src/types/Meeting.ts

```diff
@@ -4,6 +4,22 @@ export enum MeetingType {
   LEC, LAB, RES, INS, EXAM, SEM, PRAC, REC, CLD, CMP, PRL, CLAS, INT
 }
 
+export const MeetingTypeDescription = new Map<number, string>([
```

Comment:
I think the type of the key should be `MeetingType` rather than `number`

Reply:
Done.